n2y/main.py

```diff
@@ -61,67 +65,83 @@ def name_column_valid(raw_rows, name_column):
     first_row_flattened = simplify.flatten_database_row(raw_rows[0])
 
     def available_columns():
-        return filter(lambda c: isinstance(first_row_flattened[c], str),
-                      first_row_flattened.keys())
+        return filter(
+            lambda c: isinstance(first_row_flattened[c], str),
+            first_row_flattened.keys())
 
     # make sure the title column exists
     if name_column not in first_row_flattened:
-        print(f"Database does not contain the column \"{name_column}\". "
-              f"Please specify the correct name column using the --name-column flag.",
-              file=sys.stderr)
-        print("Available column(s): ",
-              # only show columns that have strings as possible options
-              ", ".join(available_columns()), file=sys.stderr)
-        return False
+        raise NotImplementedError("%s%s%s" % (
```

Comment:
I wouldn't recommend using the `%` operator like this. The implicit string concatenation used in the original version is cleaner IMO.